app/components/stepper-signup.hbs

```diff
@@ -45,18 +45,39 @@
         @isValid={{this.isValid}} 
         @setIsValid={{this.setIsValid}} 
       />
+      {{else if (eq this.currentStep 7)}}
+        <IdentitySteps::StepOne 
+        @startHandler={{this.startHandler}} 
+        @setIsPreValid={{this.setIsPreValid}}
+        @isValid={{this.isValid}} 
+        @setIsValid={{this.setIsValid}} 
+        />
+      {{else if (eq this.currentStep 8)}}
+        <IdentitySteps::StepTwo 
+        @startHandler={{this.startHandler}} 
+        @setIsPreValid={{this.setIsPreValid}}
+        @isValid={{this.isValid}} 
+        @setIsValid={{this.setIsValid}} 
+        />
+      {{else if (eq this.currentStep 9)}}
+        <IdentitySteps::StepThree 
+        @startHandler={{this.startHandler}} 
+        @setIsPreValid={{this.setIsPreValid}}
+        @isValid={{this.isValid}} 
+        @setIsValid={{this.setIsValid}} 
+        />
     {{/if}}
   </form>
 
-  {{#if (not (or (eq this.currentStep 0) (eq this.currentStep 1) (eq this.currentStep 2)))}}
-    <div class='stepper__buttons'>
-      {{#if (eq this.currentStep 6)}}
-        <Reusables::Button @text='Previous' @variant='light' @onClick={{this.decrementStep}} @test='prev' @type='button' />
-        <Reusables::Button @text='Submit' @variant='dark' @onClick={{this.startHandler}} @test='submit' @type='button' />
-      {{else}}
-        <Reusables::Button data-test-prev-btn @text='Previous' @variant='light' @onClick={{this.decrementStep}} @test='prev' @type='button' />
-        <Reusables::Button data-test-next-btn @text={{if (eq this.currentStep 5) 'Preview' 'Next' }} @variant='dark' @disabled={{if (or this.preValid this.isValid) false true}} @onClick={{this.incrementStep}} @test='next' @type='submit' />
-      {{/if}}
-    </div>
-  {{/if}}
+ {{#if (and (gte this.currentStep 3) (lte this.currentStep 6))}}
+  <div class='stepper__buttons'>
+    {{#if (eq this.currentStep 6)}}
+      <Reusables::Button @text='Previous' @variant='light' @onClick={{this.decrementStep}} @test='prev' @type='button' />
+      <Reusables::Button @text='Submit' @variant='dark' @onClick={{this.startHandler}} @test='submit' @type='button' />
+    {{else}}
+      <Reusables::Button data-test-prev-btn @text='Previous' @variant='light' @onClick={{this.decrementStep}} @test='prev' @type='button' />

```

Comment:
no need to add `data-test-prev-btn` attribute  you're already passing in @test prop

Reply:
done